react/src/components/ServiceLauncherModal.tsx

```diff
@@ -313,132 +384,314 @@ const ServiceLauncherModal: React.FC<ServiceLauncherProps> = ({
       .then((values) => {
-        if (endpoint) {
-          if (baiClient.supports('modify-endpoint')) {
-            const mutationVariables = {
-              endpoint_id: endpoint?.endpoint_id || '',
-              props: {
-                resource_slots: JSON.stringify({
-                  cpu: values.resource.cpu,
-                  mem: values.resource.mem,
-                  ...(values.resource.accelerator > 0
-                    ? {
-                        [values.resource.acceleratorType]:
-                          values.resource.accelerator,
-                      }
-                    : undefined),
-                }),
-                resource_opts: JSON.stringify({ shmem: values.resource.shmem }),
-                // FIXME: temporally convert cluster mode string according to server-side type
-                cluster_mode:
-                  'single-node' === values.cluster_mode
-                    ? 'SINGLE_NODE'
-                    : 'MULTI_NODE',
-                cluster_size: values.cluster_size,
-                desired_session_count: values.desiredRoutingCount,
-                ...getImageInfoFromInputInEditing(
-                  checkManualImageAllowed(
-                    baiClient._config.allow_manual_image_name_for_session,
-                    values.environments?.manual,
-                  ),
-                  values,
-                ),
-                name: values.serviceName,
-                resource_group: values.resourceGroup,
-              },
-            };
-            commitModifyEndpoint({
-              variables: mutationVariables,
-              onCompleted: (res, errors) => {
-                if (errors && errors?.length > 0) {
-                  const errorMsgList = errors.map((error) => error.message);
-                  for (let error of errorMsgList) {
-                    message.error(error, 2.5);
-                  }
-                } else {
-                  const updatedEndpoint = res.modify_endpoint?.endpoint;
-                  message.success(
-                    t('modelService.ServiceUpdated', {
-                      name: updatedEndpoint?.name,
-                    }),
-                  );
-                  onRequestClose(true);
-                }
-              },
-              onError: (error) => {
-                if (error.message) {
-                  message.error(error.message);
-                } else {
-                  message.error(t('modelService.FailedToUpdateService'));
+        // FIXME: temporally insert vfolderName when validation
+        values.vFolderName = endpoint?.model as string;
+        mutationsToValidateService.mutate(values, {
+          onSuccess: (data: any) => {
+            setValidationTime(validationDateTime);
+            setContainerLogSummary('loading...');
+            const response = data;
+            const sse: EventSource =
+              baiClient.maintenance.attach_background_task(response['task_id']);
```

Comment:
We handle background tasks using `upsertNotification`. Please replace SSE codes with it.
Here is the using example.
https://github.com/lablup/backend.ai-webui/blob/main/react/src/components/ContainerRegistryList.tsx

Reply:
For now, I presume that there's no action adding when receiving `bgtask_update` in upsertNotification component. Since additional action (such as showing container logs) is needed when validation, I think it's better to leave as is and generalize the upsertNotification afterward as a new issue.